Source/DafnyStandardLibraries/src/DafnyStdLibs/Base64.dfy

```diff
@@ -0,0 +1,1465 @@
+/*******************************************************************************
+ *  Copyright by the contributors to the Dafny Project
+ *  SPDX-License-Identifier: MIT
+ *******************************************************************************/
+
+/** An encoder and decoder for the Base64 encoding scheme. */
+module DafnyStdLibs.Base64 {
```

Comment:
Could you please add two exports sets, one with the functions and methods we think it's likely users will want (the default export set), and one with everything so that it could be reasoned about externally?

Reply:
I _almost_ did this before you asked. There certainly are a lot of definitions in this module. ;)